### PR TITLE
Theme

### PR DESCRIPTION
**Pull request by frayeralex**
Setup Boyko/dada theme

### CODE REVIEW

File: app/design/frontend/Boyko/dada/Magento_Newsletter/templates/subscribe.phtml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+ */
+
+// @codingStandardsIgnoreFile
+
+/** @var \Magento\Newsletter\Block\Subscribe $block */
+
+?>
+<div class="block newsletter">
+    <div class="title" style="display: block;">
+        <strong><?= $block->getTitle() ?></strong>
```
Comment (chabanenk0) [inline app/design/frontend/Boyko/dada/Magento_Newsletter/templates/subscribe.phtml]: It's unclear which changes were made in each file
In the case some original Magento files are being overwritten, Please commit the file with two commits: first one with the original content and the next one with your changes

File: app/design/frontend/Boyko/dada/Magento_Theme/layout/default.xml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            </arguments>
+        </referenceBlock>
+        <referenceContainer name="div.sidebar.additional">
+            <block
+                class="Magento\Framework\View\Element\Template"
+                name="dada.template"
+                as="dada_discount_banner"
+                template="Magento_Theme::dada/discount_banner_template.phtml">
+                <arguments>
+                    <argument name="banner_img_src" xsi:type="string">https://banner2.kisspng.com/20180626/lrq/kisspng-garage-sale-sales-clip-art-sale-10-5b32fd3029f2d4.7976958815300682721718.jpg</argument>
```
Comment (chabanenk0) [inline app/design/frontend/Boyko/dada/Magento_Theme/layout/default.xml]: Please try saving this file to your theme (in 
Try to use 
```
<img src='<?php echo $this->getViewFileUrl('images/banner-image.png'); ?>' 
```
or 
```
<img src='<?php echo $this->getViewFileUrl('Magento_Theme::images/banner-image.png'); ?>' 
```
in your template. 
Reference: https://magento.stackexchange.com/questions/93688/add-a-static-image-on-phtml-page-and-cms-blocks-magento2

File: app/design/frontend/Boyko/dada/Magento_Newsletter/templates/subscribe.phtml (hunk 1 above)
Comment (frayeralex) [reply app/design/frontend/Boyko/dada/Magento_Newsletter/templates/subscribe.phtml]: ok

File: app/design/frontend/Boyko/dada/Magento_Theme/layout/default.xml (hunk 2 above)
Comment (frayeralex) [reply app/design/frontend/Boyko/dada/Magento_Theme/layout/default.xml]: done

**Pull request merged**